en/companion_computer/holybro_pixhawk_jetson_baseboard.md

```diff
@@ -0,0 +1,1074 @@
+- Ethernet cable
+- Mouse and keyboard (the baseboard has 4 USB ports exposed from Jetson, to of which are USB 3.0)
+
+## Flashing the Jetson Board
+
+::: info
+This Jetson setup is tested with Nvidia Jetpack 6.0 (Ubuntu 22.04 base) and ROS 2 Humble, which are the versions currently supported by PX4-Autopilot community.
+The host computer is also running Ubuntu 22.04.
+:::
+
+The Jetson companion computer can be flashed from a development computer using the [Nvidia SDK Manager](https://docs.nvidia.com/sdk-manager/download-run-sdkm/index.html#download-sdk-manager) when the board is in recovery mode.
+There are many ways to put Jetson boards into recovery mode, but on this board the best way is to use the small sliding switch provided for this purpose.
+You also need to connect the development computer to the baseboard using the specific USB-C port shown in the image below.
+
+![USB port and bootloader switch](../../assets/companion_computer/holybro_pixhawk_jetson_baseboard/flashing_usb.png)
+
+::: tip
+The USB port is only for flashing the Jetson, and cannot be used as debug interface.
+:::
+
+Download [Nvidia SDK Manager](https://docs.nvidia.com/sdk-manager/download-run-sdkm/index.html#download-sdk-manager) using either the online or offline installer (you need to have an Nvidia account to install and use the SDKManager).
+
+After starting SDKManager you should see a screen similar to the one below (if the board is connected to host computer in recovery mode):
+
+![SDK Manager init page](../../assets/companion_computer/holybro_pixhawk_jetson_baseboard/nvidia_sdkmanager_1.png)
+
+The next page allows you to choose the components to be installed.
+Select the _Jetson Linux_ target components as shown (there is no point selecting other components because after flashing the board will disconnect).
+
+![SDK Manager installation components page](../../assets/companion_computer/holybro_pixhawk_jetson_baseboard/nvidia_sdkmanager_2.png)
+
+Next confirm your selected device:
+
+- Choose `Pre-config` for the OEM Configuration (this will skip Ubuntu first time setup screens after reboot).
+- Choose your preferred username and password (and write them down).
+  These will be used as your login credentials to Jetpack.
+- Choose `NVMe` as the storage device because the board has separate SSD for storage.
+
+![SDK Manager installation storage and OEM config page](../../assets/companion_computer/holybro_pixhawk_jetson_baseboard/nvidia_sdkmanager_3.png)
+
+Press **Flash** to start installing the image.
+It will take several minutes for the installation to complete.
+
+::: warning
+The fan will be running while the installation is going on, so you may make sure it not blocked.
+Jetson will boot into initial login after flashing.
+:::
+
+## Jetson Network Setup
+
+After flashing the Jetson it will reboot to the login screen (skipping recovery mode).
```

Comment:
@farhangnaderi Please:
1. Confirm that I login using the credentials I set when using the SDK manager
2. Should I disconnect the USB here? Should I slip the recovery mode swtich? If not, exactly where in the process does that happen?